client/app/validations/submittable-pas-form.js

```diff
@@ -6,6 +9,7 @@ export default {
   ...SaveablePasForm,
   dcpRevisedprojectname: [
     ...SaveablePasForm.dcpRevisedprojectname,
+    validatePresence({ presence: true, message: 'This field is required' }),
```

Comment:
super tiny nit: line break

Reply:
@godfreyyeung @allthesignals @trbmcginnis was going to bring this up during our tooling discussion, but what are your thoughts on trying an auto code formatter (prettier) so that we can standardize things like this and not think about them?